session.go

```diff
@@ -327,7 +334,7 @@ func (s *session) Retry() error {
 			return errors.Trace(err)
 		}
 		retryCnt++
-		if (s.maxRetryCnt != unlimitedRetryCnt) && (retryCnt >= s.maxRetryCnt) {
+		if !s.unlimitedRetryCount && (retryCnt >= maxCnt) {
 			return errors.Trace(err)
 		}
 		kv.BackOff(retryCnt)
```

Comment:
Usually, large transaction takes longer time to execute, adding backoff time doesn't make much difference.